api/src/main/java/org/pmiops/workbench/notebooks/LeonardoNotebooksClientImpl.java

```diff
@@ -36,6 +35,7 @@
 
   private static final String CLUSTER_LABEL_AOU = "all-of-us";
   private static final String CLUSTER_LABEL_CREATED_BY = "created-by";
+  private static final String WORKSPACE_CDR = "WORKSPACE_CDR";
```

Comment:
nit: Can you make a little enum class for the env var names? Then you could have `NotebookEnvironmentVariables.WORKSPACE_CDR` etc for clarity. I'd do this even if there's only one and you're only using it once.